vms/job/templates/job/list.html

```diff
@@ -7,19 +7,188 @@
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Name{% endblocktrans %}" type="text" name="name" value="{% if form.name.value %}{{ form.name.value }}{% endif %}">
+                        <p class="help-block">
+                            <strong>
+                                        {% for error in form.name.errors %}
+                                            {{ error }}
+                                        {% endfor %}
+                                    </strong>
+                        </p>
+                    </div>
+                </div>
+                {% else %}
+                <div class="form-group">
+                    <label class="col-md-2 control-label">{% trans "Name" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Name{% endblocktrans %}" type="text" name="name" value="{% if form.name.value %}{{ form.name.value }}{% endif %}">
+                    </div>
+                </div>
+                {% endif %}
+                {% if form.start_date.errors %}
+                <div class="form-group has-error">
+                    <label class="col-md-2 control-label">{% trans "Start Date" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Start Date{% endblocktrans %}" type="text" name="start_date" value="{% if form.start_date.value %}{{ form.start_date.value }}{% endif %}">
+                        <p class="help-block">
+                            <strong>
+                                        {% for error in form.start_date.errors %}
+                                            {{ error }}
+                                        {% endfor %}
+                                    </strong>
+                        </p>
+                    </div>
+                </div>
+                {% else %}
+                <div class="form-group">
+                    <label class="col-md-2 control-label">{% trans "Start Date" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Start Date{% endblocktrans %}" type="text" name="start_date" value="{% if form.start_date.value %}{{ form.start_date.value }}{% endif %}">
+                    </div>
+                </div>
+                {% endif %}
+                {% if form.end_date.errors %}
+                <div class="form-group has-error">
+                    <label class="col-md-2 control-label">{% trans "End Date" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by End Date{% endblocktrans %}" type="text" name="end_date" value="{% if form.end_date.value %}{{ form.end_date.value }}{% endif %}">
```

Comment:
Hi @anjali-dhanuka. Can you just put required format of the date as a placeholder? Or better if a calendar opens up.

Reply:
Calender opens up.